### PR TITLE
Pururin: Fixes 'No Pages Found' issue

### LINKED ISSUE #8198: Pururin: Domain changed and pages are encoded
### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.11.1](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [x] I have updated all installed extensions.
- [x] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

### Tachiyomi version

Stable 0.11.1

### Android version

Android 11

### Source information

Pururin 1.2.3

### Steps to reproduce

1. Open Pururin
2. Read Doujin

### Expected behavior

Can Read Doujin

### Actual behavior

When I want to read doujin It say 'No Pages Found', It happen after the Website got fix from being Down, It need an Update

### Other details

Basically Entire Doujin from Pururin Cant be read, Everytime you click read it say That
Screenshot_20210718-053609_Tachiyomi

### PR DESCRIPTION
**Pull request by Arraiment**
Should fix issue where page lists are not parsed due to changes in the website code. I've changed the jsoup implementation to reflect that. Also, the domain is updated.

Closes #8198

### CODE REVIEW

File: src/en/pururin/src/eu/kanade/tachiyomi/extension/en/pururin/Pururin.kt [hunk 1]
```diff
@@ -64,7 +64,7 @@ class Pururin : ParsedHttpSource() {
         manga.title = infoElement.select("h1").text()
         manga.author = infoElement.select("tr:has(td:contains(Artist)) a").attr("title")
         manga.artist = infoElement.select("tr:has(td:contains(Circle)) a").text()
-        manga.status = SManga.COMPLETED
+        manga.status = 2 // Completed
```
Comment (CarlosEsco) [inline src/en/pururin/src/eu/kanade/tachiyomi/extension/en/pururin/Pururin.kt:67]: Why did you change this?

File: src/en/pururin/src/eu/kanade/tachiyomi/extension/en/pururin/Pururin.kt [hunk 2]
```diff
@@ -135,13 +135,13 @@ class Pururin : ParsedHttpSource() {
...
 
-    override fun pageListRequest(chapter: SChapter): Request = GET(
+/*    override fun pageListRequest(chapter: SChapter): Request = GET(
         "$baseUrl${chapter.url.let {
             it.substringAfterLast("/").let { titleUri ->
                 it.replace(titleUri, "01/$titleUri")
             }.replace("gallery", "read")
         }}"
-    )
+    )*/
```
Comment (CarlosEsco) [inline src/en/pururin/src/eu/kanade/tachiyomi/extension/en/pururin/Pururin.kt:144]: remove this if its not used anymore

File: src/en/pururin/src/eu/kanade/tachiyomi/extension/en/pururin/Pururin.kt (hunk 1 above)
Comment (Arraiment) [reply src/en/pururin/src/eu/kanade/tachiyomi/extension/en/pururin/Pururin.kt:67]: Was trying to fix problem where every manga description was "unknown status". I realise now that there's more issues with the description parsing, I'll try fixing it first.

File: src/en/pururin/src/eu/kanade/tachiyomi/extension/en/pururin/Pururin.kt (hunk 2 above)
Comment (Arraiment) [reply src/en/pururin/src/eu/kanade/tachiyomi/extension/en/pururin/Pururin.kt:144]: Will do

File: src/en/pururin/src/eu/kanade/tachiyomi/extension/en/pururin/Pururin.kt (hunk 1 above)
Comment (Arraiment) [reply src/en/pururin/src/eu/kanade/tachiyomi/extension/en/pururin/Pururin.kt]: The above issue is on my end, I've reverted the changes.

**Pull request merged**